src/components/Panels/index.js

```diff
@@ -0,0 +1,125 @@
+import { Fragment, h } from "preact";
+import { useState } from "preact/hooks";
+import { MoreVertical as MoreVerticalIcon } from 'preact-feather';
+import {
+	ChevronDown,
+	MessageSquare,
+	AlertCircle,
+	CheckCircle,
+	Circle,
+	PauseCircle,
+} from "preact-feather";
+
+// NOT YET IMPLEMENTED
```

Comment:
what means NOT YET IMPLEMENTED

Reply:
Panel menu, is a part of a bigger refactoring : Generic Panel component to remove duplicate code in each specific panel.